warrior/Actions/SeleniumActions/browser_actions.py

```diff
@@ -184,12 +186,16 @@ def browser_launch(self, system_name, browser_name="all", type="firefox",
                 if browser_inst:
                     browser_fullname = "{0}_{1}".format(system_name, browser_details["browser_name"])
                     output_dict[browser_fullname] = browser_inst
-                    if "url" in browser_details and browser_details["url"]\
-                            is not None:
-                        result, url = self.browser_object.check_url(browser_details["url"])
-                        if result == True:
-                            result = self.browser_object.go_to(url,
-                                                               browser_inst)
+                    url = browser_details["url"]
+                    if url is not None:
```

Comment:
Hi @brettleehari,

I want to list down usecases of what should fail and what should not:

1. www.google.com: Pass
2. https://www.google.com: Pass
3. file:///path/to/file.html: Pass if file exists
4. file:///path/to/file.html: Fail if file does not exist
5. httpz://www.google.com: Fail
6. http://wwww.google.com: Fail
7/ http://googsle.com: Fail

Reply:
@brettleehari  what is the status of the above comments?

Reply:
We had a call and explained to Sanika that we cannot do the operation as mentioned in sheet with examples. 
Addressed all the concerns.